source/default/_patterns/02-molecules/carousel/_carousel.twig

```diff
@@ -50,3 +50,5 @@
     {% endblock controls %}
   </div>
 </div>
+
```

Comment:
Did you mean to add extra spaces at end of file here?

Reply:
Fixed!